examples/react-app/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@cloudscape-design/components": "^3.0.222",
+    "@cloudscape-design/components": "^3.0.126",
```

Comment:
I see the public npm version cloudscope  is 3.0.220. But why are we downgrading it to much older version?

Reply:
This is the version currently working with the @iot-app-kit/dashboard package. I'm just making them consistent. Upgrading polaris isn't really in scope for my current work, so picking a safe version.